deployment/ccip/operations/evm/v1_6/ops_fee_quoter.go

```diff
@@ -0,0 +1,188 @@
+package v1_6
+
+import (
+	"errors"
+	"fmt"
+	"math/big"
+
+	"github.com/Masterminds/semver/v3"
+	"github.com/ethereum/go-ethereum/common"
+
+	"github.com/smartcontractkit/chainlink-ccip/chains/evm/gobindings/generated/v1_6_0/fee_quoter"
+	cldf "github.com/smartcontractkit/chainlink-deployments-framework/deployment"
+	"github.com/smartcontractkit/chainlink-deployments-framework/operations"
+
+	"github.com/smartcontractkit/chainlink/deployment"
+	"github.com/smartcontractkit/chainlink/deployment/ccip/shared"
+	"github.com/smartcontractkit/chainlink/deployment/ccip/shared/deployergroup"
+	"github.com/smartcontractkit/chainlink/deployment/ccip/shared/opsutil"
+	"github.com/smartcontractkit/chainlink/deployment/ccip/shared/stateview"
+	commoncs "github.com/smartcontractkit/chainlink/deployment/common/changeset"
+	"github.com/smartcontractkit/chainlink/deployment/common/proposalutils"
+)
+
+type DeployFeeQInput struct {
+	Chain    uint64
+	Params   FeeQuoterParams
+	LinkAddr common.Address
+	WethAddr common.Address
+}
+
+var (
+	DeployFeeQuoterOp = operations.NewOperation(
+		"DeployFeeQuoter",
+		semver.MustParse("1.0.0"),
+		"Deploys FeeQuoter 1.6 contract on the specified evm chain",
+		func(b operations.Bundle, deps opsutil.OpDependencies, input DeployFeeQInput) (common.Address, error) {
```

Comment:
Can we use regular functions instead of anonymous?
It will remove nested code and improve readability and maintainability.